src/app/pages/administration/search/search-view.jsx

```diff
@@ -66,6 +67,11 @@ const useStyles = makeStyles({
     display: 'grid',
     gridTemplateRows: '3fr 1fr',
   },
+  searchButtonContainer:{
+    display:'flex',
+    justifyContent: 'space-between',
+    gridGap:'8px',
+  },
```

Comment:
```suggestion
    display: 'flex',
    justifyContent: 'space-between',
    gridGap: '8px',
  },
```

Is your linter running? It should catch these kinds of things